src/main/java/BMO/Bmo.java

```diff
@@ -0,0 +1,146 @@
+package bmo;
+
+import java.util.Scanner;
+
+import bmo.task.Task;
+
+/**
+ * Bmo is a chatbot that helps users manage their tasks.
+ */
+public class Bmo {
+
+    private static String filePath = "ip/data/BMO.txt";
```

Comment:
could possibly set this as final as i dont think it would be changing throughout the program 👍

Reply:
I agree! Better OOP 🕺🏼